rules/rbe_repo/version_check.bzl

```diff
@@ -60,4 +60,7 @@ def _check_bazel_version(bazel_version_fallback):
                "was not defined explicitly in rbe_autoconfig target. " +
                "Falling back to '%s'") % bazel_version_fallback)
         return bazel_version_fallback
+    if native.bazel_version > bazel_version_fallback:
+        print("\nCurrent running Bazel is ahead of bazel-toolchains repo. " +
+              "Please update your pin to bazel-toolchains repo.")
```

Comment:
Thanks for doing this! This should help with some of the confusion we've seen.

Maybe add "in your WORKSPACE file" or something similar?
I know it obviously follows from "a pin to a repo", but for those not intimately familiar with Bazel it's not always top of mind. And/or add a link to your Releases page that has the snippet to paste in the WORKSPACE?

Reply:
done added the "in your WORKSPACE" will not add any links to release page here to keep the message as short and as unobtrusive as possible (also don't want to make that URL part of our API surface).